src/entry-points/popup/App.svelte

```diff
@@ -370,6 +370,7 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
     return min < x && x < max;
   }
   $: timeSavedPlaybackRateEquivalents = getTimeSavedPlaybackRateEquivalents(latestTelemetryRecord);
+  $: estimatedRemainingDuration = mmSs((r?.elementRemainingDuration / timeSavedPlaybackRateEquivalents[0]) / settings?.soundedSpeed || 0);
```

Comment:
Hmm. So you're not dividing by just `timeSavedPlaybackRateEquivalents[1]`, because it's accumulative and `settings?.soundedSpeed` is instantaneous? Clever.

However, division by 0 doesn't make sense. I'd default to 1, or it'd be probably best to not show the value at all util the settings have loaded.
Oh, and another option is to divide by `timeSavedPlaybackRateEquivalents[1]` until the settings have loaded.

Reply:
Due to the order of precedence, the default to 0 was actually to make the entire equation 0 if any term was undefined or NaN. But that doesn't make sense either since `mmSs(0)` would show "00:00" for the estimated remaining duration even if there was no video playing.

I will change that so it checks for the settings being loaded and sets `estimatedRemainingDuration` to undefined otherwise.

As for the Infinity thing, I would suggest hiding it because it's just not useful for the end user.